### ISSUE TITLE
Add option to exclude certain n-grams

### ISSUE THREAD

**Issue by stefanobellelli** [stefanobellelli/nonce3vec#6]
It should be possible to discard (i.e. not count when building vectors) arbitrarily-chosen n-grams.

**Comment by stefanobellelli** [comment]
Now it is possible to exclude certain n-grams at will.

However, now POS n-grams can only be discarded in bulk, and not by the words they stand for. For example, one can discard all VBZ_DT, but can _cannot_ discard solely those POS n-grams that stand for the word n-grams "is_a" or the like.

**Comment by minimalparts** [comment]
Ah... okay. I thought spacy had specific tags for auxiliaries but I see it is not the case. Then that's actually good news because the pattern is unlikely to affect the POS tag experiments (since it presumably stands for many more combinations). 

I would then run on the *word* bigrams removing is-a patterns and see what happens there. Just to be sure.

**Comment by stefanobellelli** [comment]
So... are we actually interested in developing a more precise feature to block the POS n-grams that stand for `is_a` word n-grams? Because it would take a day or so to implement.

**Comment by minimalparts** [comment]
No, no. I simply mean that when you run the SVM on word bigrams rather than POS, you can exclude is-a then. Does that make sense?

**Comment by stefanobellelli** [comment]
Yeah I got that, it's already possible (just add the n-grams you'd like to discard in `Conf.exclude`).
